docs/user-guide/internationalization.md

```diff
@@ -149,13 +149,20 @@ component's `.ts` file:
 
 <!-- {% endraw %} -->
 
-## Adding your own messages
+## Adding and replacing messages
 
 The built-in translations certainly won't cover everything you will need for
-your app but you can easily replace them with your own lists. This involves
-making copies of the existing lists in your app's folder and adding your
-own keys. See the [Translation service](../core/translation.service.md) page for
-full details and examples.
+your app but you can easily replace them with your own lists. This enables you
+to add new keys and also replace the text of existing keys with your own.
+
+To override the default translations, you need to copy the existing source files
```

Comment:
The process is slightly different. The developers define their application-level `en.json` or any other locale file that backs the application. That file is loaded the last on the top of everything that comes from ADF. That allows creating same JSON structures as in the ADF, but inside the application locale, to overwrite the existing.

For example:
`app/path/en.json`:

```json
{
  "title": "my app",
  "LOGIN": {
     "LABEL": {
        "LOGIN": "Custom Sign In"
     }
  }
}
```

Where the `LOGIN` structure is taken from the core:
https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/core/i18n/en.json#L169